pkgs/development/interpreters/clojure/lumo.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, unzip }:
+
+stdenv.mkDerivation rec {
+  name = "lumo-${version}";
+  version = "1.8.0-beta";
+
+  src = fetchurl {
+    url = "https://github.com/anmonteiro/lumo/releases/download/${version}/lumo_linux64.zip";
+    sha256 = "0p06994w48pbgy8xwc1sz3gg609ardsdhmjafdf7qk4gclyiqs5i";
+  };
+
+  buildInputs = [ unzip ];
+
+  phases = [ "unpackPhase" "installPhase" ];
```

Comment:
Please do not set phases explicitly, rather disable phases that are causing problems, if any.

Reply:
:shipit:

Reply:
:+1: